generators/client/templates/gulpfile.js

```diff
@@ -125,44 +124,40 @@ gulp.task('styles', [<% if(useSass) { %>'sass'<% } %>], function () {
         .pipe(browserSync.reload({stream: true}));
 });
 
-gulp.task('inject', function () {
+gulp.task('inject', ['inject:app','inject:test', 'inject:vendor']);
```

Comment:
i think we should have two tasks for this

`gulp.task('inject:dep', ['inject:test', 'inject:vendor']);` and 
`gulp.task('inject:app', ['inject:app']);`